### DISCUSSION TITLE
Text Label Cutting Off in Python-Generated Diagram in VS Code

### DISCUSSION THREAD

**Discussion by asdmd12** [microsoft/vscode-discussions#2414] [Extension Development QnA] [answered]
Hi everyone,

I'm working on a network diagram using Graphviz and python code in VS Code. The issue I'm facing is that the text label of some of my nodes is being cut off or missing some pixels at the top in the output image (png). Could I kindly get some information on how to go about resolving this issue?

 Attached a file of what that looks like.
![server](https://github.com/user-attachments/assets/fad37410-d6b6-403d-9a7e-efc5e5616848)

**Comment by isidorn** [comment, marked as answer]
This discussion repo is for VS Code extension authors, not for users. For that question please try out on stack overflow.

In case you are writing a VS Code extensions do tell us what you are trying to do, and what are the issues.
